extensions/wikia/PortableInfobox/PortableInfoboxHooks.class.php

```diff
@@ -1,8 +1,9 @@
 <?php
+
 class PortableInfoboxHooks {
 	// FIX ME: temporary implementation - styles should be included only on page with portable infobox
```

Comment:
Or consider using ResourceLoader and registering a module with Infoboxes CSS as a dependency.